File: PodcastFeed/PodcastFeed/Model/Objective-C/FeedData.m
//
//  FeedData.m
//  PodcastFeed
//
//  Created by shuo on 2021/3/11.
//

#import "FeedData.h"

@implementation Episode
@synthesize title, content, pubDate, imageURL, mediaURL;
@end

@implementation Channel
@synthesize episodes, imageURL;
@end

File: PodcastFeed/PodcastFeed/Model/FeedData.swift
//
//  FeedData.swift
//  PodcastFeed
//
//  Created by shuo on 2021/3/7.
//

import Foundation

struct Channel {
    
    let imageURLString: String
    let episodes: [Episode]
}

struct Episode {
    
    let title: String
    let description: String
    let imageURLString: String
    let pubDate: Date
    let mediaURLString: String
}
